content/livesync/models/models.textile

```diff
@@ -93,30 +92,26 @@ SELECT COALESCE(MAX(sequence_id), 0) FROM outbox;
 
-When a message with the target @sequenceID@ is reached, all of the messages from that point forward are sent to the model for processing to maintain continuity with incoming live messages.
+When a message is sent to the @sequenceID@, the model processes subsequent messages to maintain continuity with live messages. If the target @sequenceID@ is unreachable, there is insufficient message history to resume from the correct point to update the model. The SDK will attempt a re-sync. It re-syncs by calling the sync function again to acquire a newer version of the model state.
 
-If the target @sequenceID@ cannot be reached, there is insufficient message history to resume from the correct point and bring the model up to date. The SDK will try to sync again with a fresher version of the model state by calling your sync function once again.
+The amount of history available to query on the channel is determined by your "message storage":/storage-history/storage configuration on the channel. This configuration must match the @syncOptions.messageRetentionPeriod@ defined in your "@ClientOptions@":/livesync/models#options. The SDK uses this configuration option as a hint as to whether to resynchronize via the sync function and skip paginating through history when messages are expected to have expired.
 
-The amount of history available to query on the channel is determined by your Ably "message storage":/storage-history/storage configuration on the channel. This configuration must match the @syncOptions.messageRetentionPeriod@ defined in your "@ClientOptions@":/livesync/models#options. The SDK uses this configuration option as a hint as to whether to resynchronize via the sync function and skip paginating through history when messages are expected to have expired.
-
-You may wish to cache the model state from the backend endpoint called by the sync function. There is a strict requirement that the model state returned by this endpoint is no older than the message retention period configured on the channel so that the model state can be brought up to date.
-
-If there are no historical messages at all on the channel, then the model state is assumed to be brand new, having not yet undergone any changes.
+To ensure the model's state is up to date, cache the state obtained from the backend endpoint used by the sync function. The model state returned from this endpoint must not exceed the message retention period set on the channel. If no previous messages exist on the channel, the model is assumed to be a new state without any modifications.
```

Comment:
> To ensure the model's state is up to date, cache the state obtained from the backend endpoint used by the sync function

The way this is phrased now sounds to me like the solution requires the user to implement some kind of caching that is not documented or elaborated on any further. This isn't the case, the point was intended to be that if the user wants to implement a cache on their backend endpoint, there are some potential issues to be aware of.

Reply:
>The model state returned from this endpoint must not exceed the message retention period set on the channel.

Also this one isn't quite right, it should say that the state returned by the endpoint is _not any older than_ the message retention period